libr/debug/p/debug_native.c

```diff
@@ -1371,7 +1371,7 @@ static bool arm32_hwbp_del (RDebug *dbg, RBreakpoint *bp, RBreakpointItem *b) {
 #endif // PTRACE_GETHWBPREGS
 #endif // __arm
 
-#if __arm64__ || __aarch64__
+#if (__arm64__ || __aarch64__) && defined(PTRACE_GETREGSET)
```

Comment:
lgtm, but maybe we shuold find away to do this at runtime, not at compile time :? otherwise the same r2 build wont work on different linux kernels

Reply:
yeah it could be possible but it will probably require a lot of changes, as you need to re-define all PTRACE_GETRESET (but probably other defines as well) so you can use them without making the compiler complain.